examples/kubernetes/README.md

```diff
@@ -0,0 +1,25 @@
+# Example Instructions
+
+The yaml files in this example folder can be used to test out New Relic's Prometheus-based Kubernetes monitoring quickstart.
+
+1. Deploy the following yamls. Note that if you already have KSM setup in your cluster, not all of these yamls will be necessary.
+
+```
+kubectl apply -f cluster-role.yaml
+kubectl apply -f ksm-cluster-role-binding.yaml
+kubectl apply -f ksm-cluster-role.yaml
```

Comment:
why are we not leveraging the KSM chart?

Reply:
it would be more configurable for the user and we would introduce basically zero code to maintain

Reply:
Can you give a pointer? happy to update, but we need to test it out to make sure that it does all the same things the above does